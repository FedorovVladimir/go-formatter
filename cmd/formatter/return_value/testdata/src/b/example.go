package a

func returnValue(a, b int) (e int, d bool) { // want "return value"
	return a + b, a > b
}
