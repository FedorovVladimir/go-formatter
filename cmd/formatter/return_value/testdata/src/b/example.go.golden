package a

func returnValue(a, b int) (int, bool) { // want "return value"
	return a + b, a > b
}
